src/box/replication.cc

```diff
@@ -1279,6 +1319,40 @@ replicaset_find_join_master(void)
 	return leader;
 }
 
+struct replica *
+replicaset_find_join_master_cfg(void)
```

Comment:
`replicaset_find_join_master_auto()` and `replicaset_find_join_master_cfg()` can be static.

Reply:
Thanks for noticing! Fixed.